### PR TITLE
feat: add wrapper subinvoke test case

### PR DESCRIPTION
**Pull request by Niraj-Kamdar**

### CODE REVIEW

**Review by cbrzn — APPROVED** [review]

File: packages/test-cases/cases/wrappers/wasm-as/simple-subinvoke/invoke/src/index.ts [hunk 1]
```diff
@@ -1,5 +1,10 @@
...
 
-export function add(args: Args_add): string {
-  Add_Module.add
+export function add(args: Args_add): i32 {
+  let importedArgs: ImportedArgs_add = {
+    a: args.a,
+    b: args.b
+  }
+  return Add_Module.add(importedArgs).unwrap()
 }
```
Comment (Niraj-Kamdar) [inline packages/test-cases/cases/wrappers/wasm-as/simple-subinvoke/invoke/src/index.ts:10]: Why this change? I'd prefer to have a different output for invoke and subinvoke. It'll be easy to debug and spot some horrible bugs that would be passed safely due to a buggy client implementation.

Comment (cbrzn) [reply packages/test-cases/cases/wrappers/wasm-as/simple-subinvoke/invoke/src/index.ts:10]: I changed it just to make it work and have a super simple example

>  I'd prefer to have a different output for invoke and subinvoke

Gotcha, if we just use the `add` method in the `invoke` and add `1` to that result, would that be enough for you? Meaning that the `invoke` method would return the sum of two numbers + 1, and the `subinvoke` would just return the sum of two numbers

Comment (Niraj-Kamdar) [reply packages/test-cases/cases/wrappers/wasm-as/simple-subinvoke/invoke/src/index.ts:10]: Yeah, that'd work. You can call it `add_and_increament` then

**Review by dOrgJelli — APPROVED** [review]

**Comment by Niraj-Kamdar** [comment]
@dOrgJelli I had requested changes that weren't addressed yet. @cbrzn Can you create a patch PR for this?

**Comment by cbrzn** [comment]
@Niraj-Kamdar yah no worries is on my todo list
